great_expectations/data_context/store/database_store_backend.py

```diff
@@ -103,7 +103,7 @@ def __init__(  # noqa: C901 - 16
             cols.append(Column(column_, String, primary_key=True))
         cols.append(Column("value", String))
         try:
-            table = Table(table_name, meta, autoload=True, autoload_with=self.engine)
```

Comment:
This was the only occurrence of `autoload=True`. We are asked to use the `autoload_with` parameter instead, which we already do.